core/array.cpp

```diff
@@ -27,12 +27,12 @@
 /* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
-#include "array.h"
+#include "core/array.h"
```

Comment:
As mentioned on IRC, in such case it should likely be kept as `array.h` to match the current .cpp.